README.md

```diff
@@ -1,3 +1,19 @@
+## Pushing changes to npm
+After merging your fix/patch/improvement/etc.. make sure you build the package with:
+
+``npm run build``
+
+We need the dist folder to be pushed to npm so make sure to remove the dist folder from your `.gitignore` before publishing.
```

Comment:
waarom dan? Je gebruikt helemaal geen git hieronder zo te zien.

En ik zie ook geen dist folder hier in de repo staan op GitHub trouwens. Ik denk dat het niet klopt.

Reply:
maar hij moet ook niet in de repo, toch?

Reply:
wacht... moeten we niet gewoon een `.npmignore` file toevoegen met daarin de juiste dirs en files?

Reply:
Nee is enkel voor het publishen naar npm (dat gebeurd gewoon vanaf je lokale filesystem).

Reply:
> wacht... moeten we niet gewoon een `.npmignore` file toevoegen met daarin de juiste dirs en files?

Opzich geen verkeerd idee inderdaad.

[https://npm.github.io/publishing-pkgs-docs/publishing/the-npmignore-file.html](https://npm.github.io/publishing-pkgs-docs/publishing/the-npmignore-file.html)

Reply:
> Nee is enkel voor het publishen naar npm (dat gebeurd gewoon vanaf je lokale filesystem).

Ja, ik las het eerst alsof je de verandering aan de gitignore file ook zou committen. 😬